JSDemos/Demos/PivotGrid/HeaderFilter/description.md

```diff
@@ -2,8 +2,8 @@ A header filter allows a user to filter field values by including or excluding t
 
 To configure a header filter, use the global [headerFilter](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/headerFilter/) object or a field's [headerFilter](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/headerFilter/) object. This demo specifies the following properties in the global **headerFilter** object:
 
-- [allowSearch](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/headerFilter/#allowSearch)     
-Allows users to search through field values.
+- [search](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/headerFilter/search/)     
+Allows you to configure a search panel within a header filter. 
 
```

Comment:
On line 14 below:

In the last sentence, can you change "checkbox" to "check box"?

Reply:
Yes, I can.